packages/extract-react-types/src/index.js

```diff
@@ -1274,8 +1276,29 @@ function convert(path, context) {
   }
 
   const converter = converters[path.type];
-  if (!converter) throw new Error(`Missing converter for: ${path.type}`);
+
+  if (!converter) {
+    const propertySignature = path.find(p => p.isTSPropertySignature() || p.isObjectTypeProperty());
+    if (propertySignature && propertySignature.node) {
+      const leadingComments = propertySignature.node.leadingComments || [];
+      const leadingComment = leadingComments.reduce((accum, comment) => accum + comment.value, '');
+
+      /**
+       * User wishes to ignore this property, since it uses unknown syntax
+       * https://github.com/atlassian/extract-react-types/issues/141
+       */
+      if (leadingComment.includes(IGNORE_STATEMENT)) return;
+    }
+
+    throw new Error(
+      `Missing converter for: ${
+        path.type
+      }, see: https://github.com/atlassian/extract-react-types/issues/150`
+    );
+  }
+
   const result = converter(path, context);
```

Comment:
eg. `const result = converter(path, context).attachComments(path.node)` probably not realistic given this effects converter, but `const results = attachComments(path.node, converter())` ? Or two calls? I dunno.